core-concepts/state.md

```diff
@@ -146,21 +142,21 @@ export default {
 }
 ```
 
-#### Modifiable state
+#### Estado modificable
 
-If you want to be able to write to these state properties (e.g. if you have a form), you can use `mapWritableState()` instead. Note you cannot pass a function like with `mapState()`:
+Si quieres se capaz de escribir en dichas propiedades del estado (por ejemplo si tienes un formulario), puedes usar `mapWritableState()` en su lugar. Cabe aclarar que no puedes pasar una función como con `mapState()`:
```

Comment:
The translation looks good, but I think we can improve in this way: "Si quieres poder escribir en estas propiedades de estado (por ejemplo, si tienes un formulario), puedes utilizar `mapWritableState()` en su lugar. Ten en cuenta que no puedes pasar una función como con `mapState()`:" Do you agree ?

Reply:
Yes yours is more understandable, i will change it